app/Services/ScBookingService.cs

```diff
@@ -231,6 +231,12 @@ public async Task<List<int>> GetConferenceTypeIds(ScCaseSearchViewModel model)
             return await GetConferenceTypesAsync(model.CaseLocationName);
         }
 
+        // Returns booking types from the cache
+        public async Task<string[]> GetAvailableBookingTypes()
```

Comment:
You might want to change the return type to `List<string>` instead of `string[]`. The SOAP API returns a `string[]` because it's best practice in SOAP to use simple types that can be consumed by both Java and C#, but when you are working in pure C# code it's always easier to pass lists around because they come with a bunch of built-in extension methods.

Reply:
Good thinking, I just updated that 👍